h2o-algos/src/main/java/hex/ensemble/Metalearner.java

```diff
@@ -86,6 +88,8 @@ void compute() {
       Log.info("Finished training metalearner model(" + _model._parms._metalearner_algorithm + ").");
 
       _model._output._metalearner = builder.get();
+      _model._dist = _model._output._metalearner._dist;
```

Comment:
Yes, looks better if it is directly copied from metalearner.

Reply:
I'm wondering if there are other metalearner properties that we want to propagate to the SE after the training...
(not asking you do this here, just currious)